docs/dev/installer.md

```diff
@@ -0,0 +1,188 @@
+```
+
+If you want a read only Dashboard, add the `--read-only` option to the command line:
+
+```bash
+./scripts/installer install --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines --read-only
+```
+
+### Installing in a custom namespace
+
+You can install the Dashboard in the namespace of your choice, this works whatever the platform (Kubernetes / OpenShift) and install method (OpenShift Pipelines Operator / manifests).
+
+To tell the `installer` script the target namespace of your choice, add the `--namespace` option:
+
+```bash
+CUSTOM_NAMESPACE=my-namespace
+
+# for kubernetes
+./scripts/installer install --namespace $CUSTOM_NAMESPACE
+
+# for openshift / openshift pipelines operator
+./scripts/installer install --openshift --namespace $CUSTOM_NAMESPACE
+
+# for openshift / manifests
+./scripts/installer install --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines --namespace $CUSTOM_NAMESPACE
+```
+
+Add the `--read-only` to the command line that suits your platform and install method to deploy a read only Dashboard in your custom namespace.
+
+### Install ingress
+
+The `installer` script can create an ingress for your Dashboard service, to enable ingress creation add `--ingress-url` and optionally `--ingress-secret` if the ingress needs TLS support.
+
+```bash
+./scripts/installer install --ingress-url my-dashboard.my-domain.com --ingress-secret my-tls-certificate
+```
+
+## Uninstall command
+
+To uninstall the Dashboard, use the `uninstall` instead of the `install` command.
+
+**IMPORTANT NOTE:** this is VERY important when you uninstall using the `installer` script that you provide the same options that were given to the script at install time. This is needed for the script to know what was created at install time and must be deleted by the `uninstall` command.
+
+```bash
+# for kubernetes
+./scripts/installer uninstall
+
+# for openshift / openshift pipelines operator
+./scripts/installer uninstall --openshift
+
+# for openshift / manifests
+./scripts/installer uninstall --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines
+```
+
+## Build command
+
+The `installer` script can be used to build the Dashboard docker image and the YAML manifests (taking care of command line options) by using the `build` command when calling the script:
+
+```bash
+# for kubernetes
+./scripts/installer build
+
+# for openshift / openshift pipelines operator
+./scripts/installer build --openshift
+
+# for openshift / manifests
+./scripts/installer build --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines
+```
+
+This will NOT deploy the resulting manifest in the target cluster but will build and push the Dashboard docker image to [whatever was configured](./README.md#build-and-deploy-with-kustomize-and-ko) for `ko` to work with and will display the YAML manifests in the console output.
```

Comment:
```suggestion
This will NOT deploy the resulting manifest in the target cluster but will build and push the Dashboard docker image to [whichever docker repo was configured](./README.md#build-and-deploy-with-kustomize-and-ko) for `ko` to work with and will display the YAML manifests in the console output.
```